docs/act.md

```diff
@@ -11,18 +11,51 @@ Each `behaviour` specification determines how a contract method updates
 the state, and its return value, and the conditions that must be
 satisfied in order for it to be applied.
 
-Alternatively, they can be thought of an initial state and a 
-set of state transitions, determining an inductively defined 
+Alternatively, they can be thought of an initial state and a
+set of state transitions, determining an inductively defined
 state transition system.
 
 The specifications can be exported to different backends in order to
 prove that the claims that are being made hold with respect to an implementation.
 
-This document is a high level description of the syntax of 
+This document is a high level description of the syntax of
 act specifications.
 For a more formal treatment of the syntax, study the
 definitions of [RefinedAst.hs](https://github.com/ethereum/act/blob/master/src/RefinedAst.hs).
 
+## Types
+
+The types of Act are made up from three basic primitives:
+Integers, Booleans and ByteStrings. Integers are unbounded,
+with true integer operations. However, as our integer expressions
+will often represent words in the EVM, we allow ourselves a slight
+abuse of notation and denote by `uintN` integers together with the
+constraint that the value fits into a `uintN`.
```

Comment:
Is this also valid for `int`?

Reply:
yes. The semantics then is that `x : intN` is `x : int s.t. 2 ^ (N - 1) <= x < 2 ^ (N -1 )`

Reply:
will add to clarify